operator/src/main/java/oracle/kubernetes/operator/rest/RestServer.java

```diff
@@ -191,31 +212,39 @@ String getInternalHttpsUri() {
    */
   public void start(Container container) throws Exception {
     LOGGER.entering();
-    if (externalHttpsServer != null || internalHttpsServer != null) {
+    if ((externalHttpsServer != null || internalHttpsServer != null) && (!isWebhook())) {
       throw new AssertionError("Already started");
     }
     boolean fullyStarted = false;
     try {
-      if (isExternalSslConfigured()) {
+      if (!isWebhook() && isExternalSslConfigured()) {
```

Comment:
Can you please make this more OO :) ?

Instead of lots of  `if (!isWebhook())` statements, please have a method that is called and then that the operator REST or webhook implementations have content or a no-op.

Reply:
I refactored the code as suggested, please let me know if I missed something. Thanks.